CMakeLists.txt

```diff
@@ -1,21 +1,23 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 2.6.2)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1.0 FATAL_ERROR)
```

Comment:
Hi!
Using OBS for building to all distros would be great! The version limit comes from adding C++11 switch, available from CMake 3.1: `set (CMAKE_CXX_STANDARD 11)`. In other case it would have to be added manually for all possible compilers (we do releases also for Windows and OSX). Do you know what is the oldest version used in OBS? Could it not be upgraded?